src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -28,6 +28,7 @@
   </PropertyGroup>
   <!-- Common -->
   <ItemGroup>
+    <Compile Include="System\Net\Http\ReadOnlyMemoryContent.cs" />
```

Comment:
nit: My OCD wants this line to be after "ClientCertificateOption.cs" to keep things alphabetical.

Reply:
fixed